code/modules/mob/living/emote.dm

```diff
@@ -88,7 +88,59 @@
 	key = "dance"
 	key_third_person = "dances"
 	message = "dances around happily."
-	restraint_check = TRUE

```

Comment:
if they're buckled it sends the message:  "twitches feebly in time to music only they can hear."